poll-webapp/src/main/webapp/vue-app/poll-activity-stream-extension/components/CreatePollComposer.vue

```diff
@@ -17,23 +17,30 @@ along with this program; if not, write to the Free Software Foundation,
 Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 -->
 <template>
-  <div>
-    <div class="actionItem" @click="openCreatePollDrawer">
-      <div class="actionItemIcon">
-        <div class="createPollComposerIcon"></div>
-      </div>
-      <div class="actionItemDescription">
-        <div :class="`actionLabel ${createdPollIcon}`"> {{ pollActionLabel }} </div>
-        <div class="actionDescription">
-          <p>{{ pollActionDescription }}</p>
-        </div>
-      </div>
+  <v-card
+    id="createPollComposerButton"
+    class="mx-4 mb-3 px-6 py-3"
+    outlined 
+    flat 
+    hover>
+    <div
+      class="d-flex flex-row align-center"
+      @click="openCreatePollDrawer">
+      <v-icon
+        color="amber darken-1"
+        class="my-1"
```

Comment:
```suggestion

```
I think we should remove this margin. In the drawer It make the height of poll button larger in height than the kudo button. By removing it, both button have same height

Reply:
ok for me. @MayTekayaa can we take care of it please?